salt/modules/drbd.py

```diff
@@ -1,14 +1,50 @@
 # -*- coding: utf-8 -*-
 '''
-DRBD administration module
+Module to provide DRBD functionality to Salt
+
+.. versionadded:: neon
```

Comment:
just one last thing. if you could remove this line here since the module itself wasn't added in neon just the new functions. thanks

Reply:
Thanks! Removed in the new commit.
Sorry for the misunderstanding of the previous fix.